src/sql/workbench/parts/notebook/notebook.contribution.ts

```diff
@@ -125,16 +126,31 @@ registerComponentType({
  * A mime renderer component for grid data.
  * This will be replaced by a dedicated component in the future
  */
-registerComponentType({
-	mimeTypes: [
-		'application/vnd.dataresource+json',
-		'application/vnd.dataresource'
-	],
-	rank: 40,
-	safe: true,
-	ctor: MimeRendererComponent,
-	selector: MimeRendererComponent.SELECTOR
-});
+if (product.quality !== 'stable') {
```

Comment:
I'm keeping this under Insiders flag for now, and we can then decide whether to include in the upcoming release.